### PR TITLE
Bad VM tests migration from elrond go

### PR DESCRIPTION
**Pull request by matei-p**

### CODE REVIEW

File: testcommon/testInitializer_inputs.go [hunk 1]
```diff
@@ -266,7 +267,7 @@ func defaultTestArwenForContracts(
 		return nil
 	}
 
-	host := DefaultTestArwenWithGasSchedule(tb, stubBlockchainHook, gasSchedule)
+	host := DefaultTestArwenWithGasSchedule(tb, stubBlockchainHook, gasSchedule, wasmerSIGSEGVPassthrough)
```
Comment (sasurobert) [inline testcommon/testInitializer_inputs.go:270]: no passthrough - this should be false.

File: testcommon/testInitializer_inputs.go [hunk 2]
```diff
@@ -306,10 +307,14 @@ func DefaultTestArwenWithWorldMockWithGasSchedule(tb testing.TB, customGasSchedu
 // DefaultTestArwen creates a host configured with a configured blockchain hook
 func DefaultTestArwen(tb testing.TB, blockchain vmcommon.BlockchainHook) arwen.VMHost {
 	customGasSchedule := config.GasScheduleMap(nil)
-	return DefaultTestArwenWithGasSchedule(tb, blockchain, customGasSchedule)
+	return DefaultTestArwenWithGasSchedule(tb, blockchain, customGasSchedule, true)
```
Comment (sasurobert) [inline testcommon/testInitializer_inputs.go:310]: this should be false

File: testcommon/testInitializer_inputs.go [hunk 3]
```diff
@@ -306,10 +307,14 @@ func DefaultTestArwenWithWorldMockWithGasSchedule(tb testing.TB, customGasSchedu
...
 }
 
-func DefaultTestArwenWithGasSchedule(tb testing.TB, blockchain vmcommon.BlockchainHook, customGasSchedule config.GasScheduleMap) arwen.VMHost {
+func DefaultTestArwenWithGasSchedule(
+	tb testing.TB,
+	blockchain vmcommon.BlockchainHook,
+	customGasSchedule config.GasScheduleMap,
+	wasmerSIGSEGVPassthrough bool) arwen.VMHost {
```
Comment (sasurobert) [inline testcommon/testInitializer_inputs.go:317]: small indentation error here

**Review by sasurobert — DISMISSED** [review]

File: test/contracts/bad-empty/elrond.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+    "language": "clang",
+    "source_files": [
+        "/home/bogdan/go2/src/github.com/ElrondNetwork/arwen-wasm-vm/test/contracts/bad-empty/empty.c"
```
Comment (camilbancioiu) [inline test/contracts/bad-empty/elrond.json:4]: Change to relative path, without local folder names.

**Review by camilbancioiu — APPROVED** [review]

**Review by camilbancioiu — CHANGES_REQUESTED** [review]
See comment regarding local folders.

**Review by sasurobert — APPROVED** [review]

**Review by camilbancioiu — APPROVED** [review]

**Pull request merged**